components/jsonld-builder.tsx

```diff
@@ -0,0 +1,16 @@
+import Head from "next/head";
```

Comment:
Should we grouping this file? Since we have 3 files here.

Maybe `components/json-ld/builder.tsx`

Reply:
Mas @mazipan, we've grouped that file and updated the PR description.